tests/databases/test_postgres.py

```diff
@@ -320,3 +320,137 @@ def test_create_table_from_select_statement(database_table_fixture):
     expected = pd.DataFrame([{"id": 1, "name": "First"}])
     test_utils.assert_dataframes_are_equal(df, expected)
     database.drop_table(target_table)
+
+
+@pytest.mark.parametrize(
+    "parameters",
+    [
+        {
+            "merge_cols": ["sell"],
+            "source_tables_cols": ["sell", "list"],
+            "target_tables_cols": ["sell", "list"],
+            "conflict_strategy": "update",
+            "expected_df": pd.DataFrame(
+                [
+                    {"sell": 142, "list": 160},
+                    {"sell": 162, "list": 330},
+                    {"sell": 175, "list": 600},
+                ]
+            ),
+        },
+        {
+            "merge_cols": ["sell"],
+            "source_tables_cols": ["sell", "list"],
+            "target_tables_cols": ["sell", "list"],
+            "conflict_strategy": "ignore",
+            "expected_df": pd.DataFrame(
+                [
+                    {"sell": 142, "list": 160},
+                    {"sell": 162, "list": 330},
+                    {"sell": 175, "list": 540},
+                ]
+            ),
+        },
```

Comment:
Why does this function require so much set up? This doesn't feel scalable

Reply:
@dimberman 
1. I think it is simpler to understand what is happening within this test this way as opposed to moving this data in fixtures/functions.

2. I don't see how we can reduce the setup code here, since that is all the parameters that the operator requires.

3. I'm getting your point regarding the scaling of this.